app/javascript/packs/leafletMap.js

```diff
@@ -1,12 +1,19 @@
 let map;
 let routeLayer;
+let layerControl;
+// Defines the floor that is currently displayed
+let currentFloor = 0;
+// object that contains for each existing floor two layer groups, one for rooms and one for labels
+const floors = {};
```

Comment:
I'm not 100% sure about this, but I think if you make these types of comments vscode should be able to show the description if you hover over the `floors` variable in later parts of the code. (Please try it out before committing).

```suggestion
/**
 * object that contains for each existing floor two layer groups, one for rooms and one for labels
 */
const floors = {};
```